UnitySetup/UnitySetup.psm1

```diff
@@ -2704,7 +2712,8 @@ function Update-UnityPackageManagerConfig {
         [String]$SearchPath,
         [int]$SearchDepth = 3,
         [Switch]$VerifyOnly,
-        [int]$PATLifetime = 7
+        [int]$PATLifetime = 7,
+        [String]$AzureSubscription
```

Comment:
[guid] is a first class type in powershell. If you convert param to [guid] from [string] then it'll reject null, empty, and do the parsing for you automatically.

Reply:
Oh good call - updated